templates/10-elasticsearch.conf

```diff
@@ -0,0 +1,6 @@
+# {{ ansible_managed }}
+# file: elasticsearch/templates/10-elasticsearch.conf
+
+{{ elasticsearch.user }}    soft    nofile    65535
```

Comment:
Thats correct actually :-)

...if it was possible to have more that number would be elastic `best practices` :+1: